packages/styles/scss/themes/default.scss

```diff
@@ -60,5 +60,5 @@ $accent: #1f8cf9;
   --box-shadow: 0 0.5rem 1.5rem rgba(0, 0, 0, 0.15);
 
   --grid-gap: 0.5rem;
-  --control-bar-height: 62px;
+  --control-bar-height: 69px;
```

Comment:
Made this adjustment for you to fix a little bit of white space that was showing through on video conference view. The control bar's height has increased, perhaps as a result of some padding box changes? Still reviewing.